### PR TITLE
Jeffm73 richmond

### PR DESCRIPTION
**Pull request by JeffM73**

### CODE REVIEW

**Comment by JeffM73** [comment]
@githubteacher

**Comment by JeffM73** [comment]
@githubteacher this closes #3924

**Comment by JeffM73** [comment]
gdlkdalgldkd;g;

**Comment by zalew16** [comment]
P

File: _pins/Jeffm73.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubHandle: Jeffm73 
+latitude: 37.6273
+longitude: -77.5437
```
Comment (hectorsector) [inline _pins/Jeffm73.json]: @JeffM73 it looks like you've got two files. You can keep `Jeffm73.json` and should remove `Jeffm73location.json`. 

You'll also need to make sure that your code matches exactly the template in the class, including `---` before and after the code:
```
---
githubHandle: YOUR-USERNAME
latitude: YOUR_LATITUDE
longitude: YOUR_LONGITUDE
---
```

**Comment by brianamarie** [comment]
Hi @JeffM73! I'm going to go ahead and make the changes now.

**Comment by brianamarie** [comment]
Hi @JeffM73! I'm going to merge now.

**Pull request merged**